nix-rust/src/error.rs

```diff
@@ -1,30 +1,55 @@
+//! Error types
+
 use std::fmt;
 
+/// An error that can occur in this crate.
 #[derive(Debug)]
 pub enum Error {
+    /// The store path could not be resolved.
```

Comment:
IMHO this isn't really desirable since it duplicates the error descriptions in the `fmt::Display` trait implementation.

Reply:
I see what you mean. The error variants have very explicit names too, so perhaps these docs don't add much.